src/main/resources/default/templates/biz/login.html.pasta

```diff
@@ -1,6 +1,8 @@
 <i:arg type="String" name="originalUrl"/>
+<i:arg type="boolean" name="noindex"
+       default="@('/' != CallContext.getCurrent().get(WebContext.class).getRawRequestedURI())"/>
```

Comment:
is this also true for "http://host.tld"

Reply:
At least in S2 the URI is `http://host.tld`, while the `WebContext` gives me `/`

Reply:
+1